src/redux/tablesRedux.js

```diff
@@ -45,7 +43,7 @@ export const changeTableStatusInAPI = (status, id) => {
   return (dispatch, getState) => {
     dispatch(fetchStarted());
 
-    Axios.post(`${api.url}/api/${api.tables}`)
+    Axios.patch(`${api.url}/api/${api.tables}/${id}`)
```

Comment:
Zabrakło CI tu czegoś. Zerknij w mój poprzedni komentarz. Jako drugi argument musisz wysłać dane jakie json-server ma zaktualizować.

Reply:
Done. @achromik ciekawe dlaczego to działało?